static/service-worker.js

```diff
@@ -24,7 +24,15 @@ self.addEventListener('push', function(event) {
     .then(function(data) {
       var title = data ? data.title : 'Mercurius';
       var body = data ? data.body : 'Notification';
-
+      if (title === 'unregister') {
```

Comment:
In a follow-up, we should make the payload contain a `command` parameter that selects the behavior of the push event handler (with the default being `notify`)